### DISCUSSION TITLE
IntelliJ share project on GitHub stuck at adding files to git...

### DISCUSSION THREAD

**Discussion by hh8923** [community/community#141952] [New to GitHub]
Hi,

New to GitHub.  If this is the wrong forum, please move it the right place.

I have latest IntelliJ community edition installed on the Mac.  When I choose to Share Project on GitHub, it got stuck on this step.

![20241015_220009~01](https://github.com/user-attachments/assets/7eba17c1-0acd-4b54-9142-9f73437ef3fd)

I can see the repo is created but empty.

How can I fix this problem?

Thanks

**Comment by ghostinhershell** [comment]
Hi @hh8923, thanks so much for reaching out in our community 👋🏾 

This question has come up before in our community, and was solved here: https://github.com/orgs/community/discussions/128045#discussioncomment-9766193. I see some people found success reinstalling IntelliJ IDE while others found success updating the intellij. I would say try updating first and, if that doesn't work, try uninstalling and re-installing the IDE.

Let me know how that works our for you!
